### PR TITLE
Switch from Boids to Circles for a simpler example/template model

### PR DESCRIPTION
**Pull request by ptheywood**
This is part of https://github.com/FLAMEGPU/FLAMEGPU2/issues/349

### CODE REVIEW

File: model.py [hunk 1]
```diff
@@ -1,466 +1,214 @@
...
+                    pen.addVertex(x * RADIUS, y * RADIUS, ENV_MIN)
+                    pen.addVertex(x * RADIUS, y * RADIUS, DIM_MAX)
+        # Activate the visualisation.
+        visualisation.activate()
+
+    # Initialise the simulation
+    simulation.initialise(sys.argv)
+
+    # Generate a population if an initial states file is not provided
+    if not simulation.getSimulationConfig().input_file:
```
Comment (Robadob) [inline model.py]: You removed this method, I will fix it.

File: model.py [hunk 2]
```diff
@@ -1,466 +1,214 @@
...
+        # Seed the host RNG using the cuda simulations' RNG
+        random.seed(simulation.getSimulationConfig().random_seed)
```
Comment (Robadob) [inline model.py]: Same problem

File: model.py (hunk 1 above)
Comment (ptheywood) [reply model.py]: I must have dev'd this with an install from an older wheel. Well spotted.

Comment (Robadob) [reply model.py]: I was just testing two py/c examples for my dissertation meeting and found it, then saw it was present here too.

**Pull request merged**